requirements.prod.in

```diff
@@ -19,7 +19,7 @@ furl
 https://github.com/opensafely-core/pipeline/archive/refs/tags/v2023.08.09.102223.zip#egg=opensafely-pipeline
 gunicorn
 incuna-mail
-interactive_templates@https://github.com/opensafely-core/interactive-templates/archive/06ae9bc49fd536599aef77973912757594f851d3.zip
+https://github.com/opensafely-core/interactive-templates/archive/refs/tags/v2023.09.12.100407.zip#egg=interactive_templates
```

Comment:
why revert to the deprecated `#egg` syntax? AIUI, `pgk@url` is the standardised version of that?

The .txt file uses it, for example?

Reply:
I copied pipeline's definition and didn't know #egg was deprecated, TIL!